packages/react/src/components/Alert/Alert.tsx

```diff
@@ -0,0 +1,134 @@
+import * as _ from 'lodash'
+import * as PropTypes from 'prop-types'
+import * as React from 'react'
+
+import {
+  UIComponent,
+  UIComponentProps,
+  ContentComponentProps,
+  commonPropTypes,
+  customPropTypes,
+  childrenExist,
+  rtlTextContainer,
+} from '../../lib'
+import { RenderResultConfig } from 'src/lib/renderComponent'
+import { defaultBehavior } from '../../lib/accessibility'
+import { Accessibility } from '../../lib/accessibility/types'
+import { ComponentEventHandler, ReactProps, ShorthandValue } from '../../types'
+import { Box, Icon } from '../..'
+
+export type AlertSlotClassNames = Record<'content' | 'closeIcon', string>
```

Comment:
I like `Record`, but let's be consistent with other components there and use `interface`

Reply:
I will, but in this way how are we going to use new TS features? :)
have a PR for all project dedicated just to changing interfaces that have the same type for value to record?

Reply:
Personally I think, that yes it will be better to have one PR for it. And introduce specific type, like `SlotClassNames<T>` :)

Reply:
will follow up on it, thanks!

Reply:
done, thx